add post about mentoring

NickleDave opened the pull request:
Hi all, I have finally got this post to a place where I don't hate it, so I want to go ahead and run it by you for feedback.

lwasser commented:
👋 @NickleDave and @xmnlab i'm finally starting to get caught up and saw this email in my inbox!! i'm super excited to read this post. i'll have a look and we can get it online soon!! Dave -thank you for writing this. @xmnlab thank you for both the review AND for the suggestion that we have blog pots!!! this is on my radar now to get live SOON!!

NickleDave commented:
Thank you for your review @xmnlab and for catching that typo.
And thank you for introducing me to http://opensciencelabs.org/ -- I see there's a page on mentoring; if group members would be interested in discussing more I can definitely start a topic on the Discourse.

And thank you @lwasser -- just let me know when you've had a chance to give it a look. I do not  want to add any more to your plate, but if you do have feedback I would love to hear it. And I can do one more pass through the post to polish it up.

lwasser commented on _posts/2020-03-22-worldwide-rse-mentoring-network.md:
```diff
@@ -0,0 +1,240 @@
+First, let me say I am fully aware there are plenty of great resources to help
+researchers understand best practices for organizing computational research
+projects, and for developing research software libraries.
+For example, my go-to is [Good Enough Practices in Scientific Computing]
```
this link is broken i think because the url should be `()` instead of `<>`

lwasser commented on _posts/2020-03-22-worldwide-rse-mentoring-network.md:
```diff
@@ -0,0 +1,240 @@
+For example, my go-to is [Good Enough Practices in Scientific Computing]
+<https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5480810/#sec009title>
+and I am definitely learning a lot from
+<https://merely-useful.github.io/py-rse/py-rse-package-py.html>.
```
These links will not work as formatted. i'm actually thinking that we should probably use html links here to open in a new tab if the link is outside of pyopensci.org. the html will look like this dave:

<a href="https://merely-useful.github.io/py-rse/py-rse-package-py.html" target = "_blank">text here describing link</a>

Will you please fix all of the links here that go to an outside page?

NickleDave replied:
This one I changed to parentheses instead of angle brackets and removed the line break between the `[title]` and the `(link)` so it should work now.

lwasser commented on _posts/2020-03-22-worldwide-rse-mentoring-network.md:
```diff
@@ -0,0 +1,240 @@
+I enjoy finding out how wrong I am. So here I will outline two straw man
+solutions, and I hope people will comment on them.
+
+### straw man solution number 1: the Carefully-Curated Collection of Domain-Specific Scientific Python Projects
```
```suggestion
### Straw man solution number 1: the Carefully-Curated Collection of Domain-Specific Scientific Python Projects
```

lwasser replied:
generally let's make our headers all `Title Case`

lwasser commented on _posts/2020-03-22-worldwide-rse-mentoring-network.md:
```diff
@@ -0,0 +1,240 @@
+It bothered me so much that I didn't have a good answer for that student that
+asked for examples that I made an attempt at this, that you can see in the
+PyOpenSci discourse here:
+<https://pyopensci.discourse.group/t/real-world-repo-packs-looking-for-examples/151>
```
another example of a link that won't work.

NickleDave replied:
Not sure why this one won't work? Please see reply below

lwasser commented on _posts/2020-03-22-worldwide-rse-mentoring-network.md:
```diff
@@ -0,0 +1,240 @@
+same idea on this episode of the Talk Python podcast:
+https://talkpython.fm/episodes/show/252/what-scientific-computing-can-learn-from-cs  
+I tweeted about it
+https://twitter.com/nicholdav/status/1235336475773939713
```
another link -- let's use <a href=>

lwasser commented on _posts/2020-03-22-worldwide-rse-mentoring-network.md:
```diff
@@ -0,0 +1,240 @@
+event. The group's mission makes a lot of sense to me, and lines up with
+a lot of other activities I've been involved in, like the graduate data science
+group at my home institution, Emory University.
+Which brings me to the germ of the idea for this blog post. Our group
```
do you mean germ of this idea??

lwasser commented:
@NickleDave thank you for all of your effort on this. I left a few comments. it is very close.

NickleDave commented:
Hi @lwasser thank you for taking the time to review this and for your helpful comments.

I force pushed some changes:
- put the headers in title case
- added another header to break up one lengthy section
- cleaned up some language / formatting issues

I didn't realize the force pushes would break the links to your comments; maybe I should have just made more commits.

Can you explain a little more about why the links won't work?
The angle brackets are the Markdown syntax I learned for formatting explicit links, as explained here:
https://daringfireball.net/projects/markdown/syntax#autolink
The site is in Jekyll, and Jekyll uses Markdown, right?
I'm happy to fix them, just not quite clear on the why or how.

I think I can add the excerpt and finish this up in the next couple of days.

lwasser commented:
hey there @NickleDave i built the blog post locally and many of the links were not working. 
So i've jus gone athead and update them. it may just be the flavor of markdown that jekyll reads. 

the image did not render and many of the links were not links when rendered! but also there are SO MANY links in this post it's best practice to open links that go to a new site in another window. this is because it draws users away from our site. And we want people to learn more about pyopensci! have a look at my changes and thank you for your patience!
if you are ok with it as is, we can merge it today!!

lwasser commented:
oh!! your author name isnt' there. let's add you. 
<img width="1321" alt="Screen Shot 2020-05-08 at 9 59 13 AM" src="https://user-images.githubusercontent.com/7649194/81424091-9eed1d80-9112-11ea-8b84-180bcd1cb19e.png">

Can you please add to this pr just like niels and anita did -- add your name to the authors .yml and add an image. thank you dave!! 
https://github.com/pyOpenSci/pyopensci.github.io/blob/master/_data/authors.yml

lwasser commented:
<img width="1058" alt="Screen Shot 2020-05-08 at 10 00 56 AM" src="https://user-images.githubusercontent.com/7649194/81424207-d3f97000-9112-11ea-9c64-3a8857d84317.png">
when you add yourself then you will see this on your post - like anita has!!

NickleDave commented:
@lwasser thank you so much for editing the links and adding the summary.
I should have thought to build the site locally myself and test the links.

I have now set up a dev environment and checked the post myself.
Aside: do we want to have instructions for setting up a dev environment for the site posted somewhere?

Seeing the post on the local build made it obvious to me that some of the tone needed re-working. I am still in the process of revising. I need to deal with a family situation but I think I can have the edits done by the end of the week at the latest--sorry it's taking forever. I will add myself to the authors as well.

lwasser commented:
@NickleDave i hope everything is ok with your family! No worries on the time frame. I have been very slow myself to find time to have a look at this post. it's really great. it's always trick to write something and not know how it will render on a website so I really do understand the nuances associated with submitting a post like this! 

to answer your question there are NO INSTRUCTIONS for setting up the dev environment! Goodness. I'll open an issue to add that information to our readme. Great suggestion!  
Thank you again. just ping me again when you need another review. i'm guessing we'll have this live this week!! or next. family always comes first.

NickleDave commented:
still making progress on this @lwasser -- almost there

btw, it was driving me crazy that text editors (e.g. Atom) parse the `target="_blank"` in the html tags as the start of an italics block, and then render huge chunks of text as italics. I found this workaround where you just insert an html comment to "close" the block, like so:
```
<!-- markup clean_ -->
```
https://github.com/atom/language-gfm/issues/44#issuecomment-111044333
https://github.com/leipert/language-pfm/issues/5#issuecomment-111043393

... in case that was also driving you crazy

lwasser commented:
no rush @NickleDave just let me know when you'd like me to have another look!! :) i stopped using markdown preview in atom as it was frustrating me :). i love atom but not those preview tools.

lwasser commented:
hi there @NickleDave just checking in on this post! i can build it locally and help too if you need it! :) i know it's busy times right now!

NickleDave commented:
Going to close this for now.
I might come back and split up into separate posts at some point, but as is it's just too long and I don't feel like I'm making a useful contribution to any dialogues about topics I'm discussing here.
